plone-6.0.x.cfg

```diff
@@ -20,3 +20,4 @@ robotframework-assertion-engine = 2.0.0
 robotframework-debuglibrary = 2.3.0
 robotframework-pythonlibcore = 4.2.0
 grpcio-tools = 1.59.0
+readme_renderer = 43.0
```

Comment:
Meh. Still fails. I found that it is a dependency of [`zest.releaser`](https://zestreleaser.readthedocs.io/en/latest/changelog.html#a2-2023-07-19). See https://github.com/zestsoftware/zest.releaser/blob/master/pyproject.toml#L18. You could pin `zest.releaser` to `< 9.0.0` or `= 8.0.0` and see if that avoids the issue, although it won't fix it. At some point, it will need to be addressed.

Reply:
solved with last main branch merge